Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:20 as build
```

Comment:
as => AS (I don't care, but to suppress warning in build)

Reply:
Done